style.css

```diff
@@ -3353,6 +3358,44 @@ ul.footer-social li {
 /*	17. Media Queries
 /* --------------------------------------------------------------------------------------------- */
 
+@media ( min-width: 480px ) {
+
+	/* Block: Pullquote -------------------------- */
+
+	/* STYLE: DEFAULT */
```

Comment:
This is a bit of a special case... "BLOCK: BASE ALIGNMENT WIDTH" could maybe work, as it's similar to the "BLOCK: BASE MARGINS" subheading.